tests/functional/aws-node-sdk/lib/utility/website-util.js

```diff
@@ -216,6 +217,25 @@ function _assertResponseHtmlRedirect(response, type, redirectUrl, method) {
         'Best redirect link ever');
         _assertResponseHtml(response.body, 'h1',
         'Welcome to your redirection file');
+    } else if (type.startsWith('redirect-error')) {
+        assert.strictEqual(response.statusCode,
+            type === 'redirect-error-found' ? 302 : 301);
+        assert.strictEqual(response.headers.location, redirectUrl);
+        for (const [key, val] of Object.entries(expectedHeaders)) {
+            assert.strictEqual(response.headers[key], val);
+        }
+
+        if (type === 'redirect-error-found') {
+            _assertContainsHtml(response.body);
+            _assertResponseHtml(response.body, 'title', '302 Moved Temporarily');
+            _assertResponseHtml(response.body, 'h1', '302 Moved Temporarily');
+            _assertResponseHtml(response.body, 'ul', [
+                'Code: Found',
+                'Message: Resource Found',
+            ]);
```

Comment:
`redirect-error-found` is in preparation for the other redirect error type for CLDSRV-489 it will be used in another PR